### PR TITLE
SPCTR-2958 - Fix: Separator Stylebook Removal

### PR DESCRIPTION
**Pull request by bsf-zanev**
### Description
Removed the Separator block from the Stylebook.

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
Tested as shown in [this comment](https://brainstormforce.atlassian.net/browse/SPCTR-2958?focusedCommentId=36331).

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +166,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Separator - Removed the block from the Style Book.
```
Comment (AbhijitNage123) [inline README.md:170]: Fix: Separator - Removed the block preview image from the Style Book.

Comment (bsf-zanev) [reply README.md]: Thanks, that's better - Updated the same.

File: src/blocks/separator/block.js [hunk 2]
```diff
@@ -9,8 +9,13 @@ import save from './save';
 import { __ } from '@wordpress/i18n';
 import { registerBlockType } from '@wordpress/blocks';
 import PreviewImage from '@Controls/previewImage';
+import { applyFilters } from '@wordpress/hooks';
+import addCommonDataToSpectraBlocks from '@Controls/addCommonDataToSpectraBlocks';
+let separatorCommonData = {};
```
Comment (AbhijitNage123) [inline src/blocks/separator/block.js:14]: let separatorCommonData = applyFilters( 'uagb/separator', addCommonDataToSpectraBlocks( {} ) );

Comment (bsf-zanev) [reply src/blocks/separator/block.js:14]: If we're doing this, shall we make it a constant instead?

Comment (AbhijitNage123) [reply src/blocks/separator/block.js:14]: Yes we can.

Comment (AbhijitNage123) [reply src/blocks/separator/block.js:14]: Yes we can.

Comment (bsf-zanev) [reply src/blocks/separator/block.js]: Great, updated the same.

**Review by AbhijitNage123 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Review by rajmohanreddy774 — APPROVED** [review]

**Review by rahulvarma722 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**